roles/hive/tasks/main.yaml

```diff
@@ -0,0 +1,7 @@
+- name: Setup hive
```

Comment:
do you want to add a dependency checker? go, docker and so on? mainly because hive does a lot and failure reasons are annoying to debug (as it is today). it also tends to just skip over failures and do the next thing, so you often don't realize something failed until you look for it

Reply:
Yeah, agree.... I'll update the script to fail and exit in certain conditions:
- Failed to build the hive binaries
- Failed to deploy hive web
- Failed to prune log results
- Failed to update web index
- Failed to update hive repo (git src)

Things that won't stop the application (Just because they can happen a bit too often):
- Failed docker builds
- Failed hive runs

Reply:
Re: dependencies. So if go, or docker is missing, you'll know, because the service won't be running due to being in a crash loop.

Reply:
Additionaly, we should probably setup some alerting on grafana to watch hive logs.

Reply:
That makes perfect sense :D Gonna resolve this convo and we can track the other changes elsewhere